devops/aws/roles/rpc/tasks/main.yml

```diff
@@ -22,10 +22,48 @@
     enabled: yes
   become: yes
 
+- name: Build required packages
+  shell: yarn build:packages
+  args:
+    chdir: '{{ remote_code_path }}'
+  # Run in async fashion for max duration of 1 hr
+  async: 3600
+  poll: 0
+  register: build_result
+
+- name: Check on build async task
+  async_status:
+    jid: '{{ build_result.ansible_job_id }}'
+  register: job_result
+  until: job_result.finished
+  # Max number of times to check for status
+  retries: 36
+  # Check for the status every 100s
+  delay: 100
+
+- name: Run query node containers
+  command: yarn workspace query-node-root start:dev
```

Comment:
the `.env` which is picked up by docker-compose has `JOYSTREAM_NODE_WS=ws://joystream-node:9944/` assumes the local joystream-node was run as docker-container.

we need to override this for indexer to work correctly.
So setting it to `wss://{{ inventory_hostname }}.nip.io/ws-rpc` or simply `ws://{{ inventory_hostname }}:9944/`